City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal core and module updates for 21.2.2022

**To test:**
- Checkout branch, then SSH into the container and run `composer install;drush deploy`
- Check that there aren't any uncommitted configuration changes after the database update: `drush cex -y`
- Check that there are no pending updates: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
- Test that the new "Content lock" functionality works:
  - Create a new page: https://helsinki-paatokset.docker.so/fi/node/add/landing_page
  - After saving, edit it. It should display a message about the content being locked
- Click around the site a bit to see that nothing is broken
- Check that the main functionality still works on the following pages:
  - https://helsinki-paatokset.docker.so/fi/paattajat
  - https://helsinki-paatokset.docker.so/fi/asia
  - https://helsinki-paatokset.docker.so/fi/kokouskalenteri
  - https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto

**List of updates:**
```
  - Upgrading drupal/core (9.3.5 => 9.3.6): Checking out 2c0c89a0f4 from cache
  - Upgrading sebastian/global-state (5.0.4 => 5.0.5): Checking out 0ca8db5a5f from cache
  - Upgrading phpunit/php-code-coverage (9.2.10 => 9.2.11): Checking out 665a1ac0a7 from cache
  - Upgrading phar-io/version (3.1.1 => 3.2.1): Checking out 4f7fd7836c from cache
  - Upgrading drupal/core-dev (9.3.5 => 9.3.6)
  - Upgrading drupal/hdbt (2.2.6 => 2.2.8): Checking out 5463ca40d8 from cache
  - Upgrading drupal/hdbt_admin (1.4.5 => 1.4.6): Checking out c9f375da59 from cache
  - Upgrading drupal/publication_date (2.0.0-beta3 => 2.0.0-beta4): Checking out 8.x-2.0-beta4 from cache
  - Upgrading drupal/pathauto (1.8.0 => 1.9.0): Checking out 8.x-1.9 from cache
  - Upgrading drupal/matomo (1.14.0 => 1.15.0): Checking out 8.x-1.15 from cache
  - Upgrading drupal/crop (2.1.0 => 2.2.0): Checking out 8.x-2.2 from cache
  - Upgrading ruflin/elastica (dev-master 96043fa => dev-master 6594f0e): Checking out 6594f0e4c2 from cache
  - Upgrading drupal/core-recommended (9.3.5 => 9.3.6)
  - Installing drupal/content_lock (2.2.0): Extracting archive
  - Upgrading drupal/helfi_platform_config (2.4.7 => 2.5.0): Checking out b1d3854a85 from cache
  - Upgrading consolidation/site-process (4.1.3 => 4.2.0): Checking out 9ef08d4715 from cache
  - Upgrading consolidation/robo (3.0.7 => 3.0.9): Checking out 33b2649c3b from cache
```
